packages/playwright-core/src/cli/cli.ts

```diff
@@ -309,13 +309,28 @@ Examples:
 
 if (!process.env.PW_LANG_NAME) {
   let playwrightTestPackagePath = null;
+  const resolvePwTestPaths = [__dirname, process.cwd()];
   try {
     playwrightTestPackagePath = require.resolve('@playwright/test/lib/cli', {
-      paths: [__dirname, process.cwd()]
+      paths: resolvePwTestPaths,
     });
   } catch {}
 
   if (playwrightTestPackagePath) {
+    const pwTestVersion = require(require.resolve('@playwright/test/package.json', {
```

Comment:
Site note: all of it isn't quite right it seems. Consider the case with the global installation of playwright, that led to the global installation of playwright-core, and a local installation of @playwright-test. Our logic here does not work then.